collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -630,8 +630,7 @@ private void loadFromIntent(Intent intent) {
 
             formPath = candidateForms.get(0).getFormFilePath();
         } else if (uriMimeType != null && uriMimeType.equals(FormsProviderAPI.CONTENT_ITEM_TYPE)) {
-            String projectId = uri.getQueryParameter("projectId");
```

Comment:
No need to care about anything other than the current project in here any longer. It'd be a lot of extra work to get `FormEntryActivity` to actually function properly with a form outside the current project, so I thought leaving this in felt inconsistent.